rules/windows/process_access/proc_access_win_direct_syscall_ntopenprocess.yml

```
@@ -59,6 +59,12 @@ detection:
         TargetImage|endswith: '\Discord.exe'
     falsepositives14:
         TargetImage: 'C:\WINDOWS\system32\AUDIODG.EXE'
+    falsepositives15:
+        SourceImage|startswith: 'C:\Users\'
+        SourceImage|endswith: '\AppData\Local\yammerdesktop\app-3.4.5\Yammer.exe'
+        TargetImage|startswith: 'C:\Users\'
+        TargetImage|endswith: '\AppData\Local\yammerdesktop\app-3.4.5\Yammer.exe'
```

will the app version not change in the future?
```suggestion
        SourceImage|contains: '\AppData\Local\yammerdesktop\app-'
        SourceImage|endswith: '\Yammer.exe'
        TargetImage|startswith: 'C:\Users\'
        TargetImage|contains: '\AppData\Local\yammerdesktop\app-'
        TargetImage|endswith: '\Yammer.exe'
```